src/backend/distributed/commands/multi_copy.c

```diff
@@ -852,6 +864,32 @@ CanUseBinaryCopyFormat(TupleDesc tupleDescription, CopyOutState rowOutputState)
 
 
 /*
+ * BinaryOutputFunctionIsDefined checks whether binary output function is defined
+ * for the given type.
+ */
+static bool
+BinaryOutputFunctionDefined(Oid typeId)
+{
+	Oid typeFunctionId = InvalidOid;
+	Oid typeIoParam = InvalidOid;
+	int16 typeLength = 0;
+	bool typeByVal = false;
+	char typeAlign = '0';
+	char typeDelim = '0';
```

Comment:
Could you assign char variables directly to 0? '0' is a random character.

Such as: https://github.com/citusdata/citus/blob/33b58f8e267ceabc81a764042226b70c1a856b1c/src/backend/distributed/commands/create_distributed_table.c#L249